src/main/scala/org/constellation/p2p/PeerAPI.scala

```diff
@@ -98,6 +98,16 @@ class PeerAPI(override val ipManager: IPManager[IO])(
               }
             }
           }
+        } ~
+        path("join") {
+          entity(as[HostPort]) { hp =>
+            dao.cluster.join(hp).unsafeToFuture
+            complete(StatusCodes.OK)
```

Comment:
This operation has always been successful? 
are we not catching or recording an error?

Reply:
Yes, I'd like to make it "fire and forget"

Reply:
I think we should `unsafeToFuture` only when integrating with legacy code otherwise `shift *> asyncFireForget` should be used